### PR TITLE
I added a link to the github account of each contributor

### PR DESCRIPTION
**Pull request by VladimirRs015**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -1,54 +1,80 @@
...
+  }
 }
 
 const collaborators = [
   {
     name: 'Winner Crespo',
     role: 'Propietario',
-    photo_url: 'https://media-exp1.licdn.com/dms/image/C4E03AQGBPcza7XsREQ/profile-displayphoto-shrink_200_200/0?e=1603324800&v=beta&t=4xbvuWqc5YmeYmsEXTtfNg_f9S3hzIf31LH48l8sbxc'
+    photo_url: 'https://media-exp1.licdn.com/dms/image/C4E03AQGBPcza7XsREQ/profile-displayphoto-shrink_200_200/0?e=1603324800&v=beta&t=4xbvuWqc5YmeYmsEXTtfNg_f9S3hzIf31LH48l8sbxc',
+    github: 'wistcc'
```
Comment (wistcc) [inline index.js]: `github` sounds too generic, what if we change the property name to something like `github_username` or something similar?

**Review by vetom — COMMENTED** [review]
Muy apero!!

**Review by wistcc — APPROVED** [review]
Can you add a screenshot to the PR description with how it looks like after your changes and also you have a little conflict there. :)

Thanks for the collaboration! 💪

**Comment by VladimirRs015** [comment]
liga_del_codigo_github_username_btn

**Review by vetom — APPROVED** [review]
LGTM!!

**Pull request merged**